dev/tasks/conda-recipes/pyarrow/meta.yaml

```diff
@@ -45,14 +45,18 @@ requirements:
     - arrow-cpp {{ ARROW_VERSION }}
 
   run:
+    - glog
+    - snappy
     - boost-cpp
     - python
     - setuptools
     - {{ pin_compatible('numpy', lower_bound='1.14') }}
     - pandas
     - six
     - futures  # [py27]
-    - arrow-cpp {{ ARROW_VERSION }}
+    - {{ pin_compatible('arrow-cpp') }}
```

Comment:
Pyarrow [misses](https://travis-ci.org/kszucs/crossbow/builds/511950378#L3166) libarrow, and it indeed has not installed in the test environment. I haven't used `run_exports` so far, so I could use a little help here @xhochy :)